deploy/auth/production/.env.production

```
@@ -0,0 +1,5 @@
+auth_POSTGRES_DB__HOST=shared-postgres
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The auth_POSTGRES_DB__HOST key should be in uppercase

(LowercaseKey)